### ISSUE TITLE
[Panda.tv] ERROR: unable to download video data: HTTP Error 403: Forbidden

### ISSUE THREAD

**Issue by brlala** [rg3/youtube-dl#13849]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.08.06*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [X] I've **verified** and **I assure** that I'm running youtube-dl **2017.08.06**

### Before submitting an *issue* make sure you have:
- [X] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections

### What is the purpose of your *issue*?
- [X] Bug report (encountered problems with youtube-dl)
---
Link: https://www.panda.tv/13653

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
>you -v https://www.panda.tv/13653
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.panda.tv/13653']
[debug] Encodings: locale cp936, fs mbcs, out cp936, pref cp936
[debug] youtube-dl version 2017.08.06
[debug] Python version 3.4.4 - Windows-10-10.0.15063
[debug] exe versions: ffmpeg N-86755-g0780ad9, ffprobe N-86755-g0780ad9
[debug] Proxy map: {}
[PandaTV] 13653: Downloading JSON metadata
[debug] Default format spec: best
[debug] Invoking downloader on 'https://pl8.live.panda.tv/live_panda/a81147d7f2f1cf8d46a7daef1e636fc5.flv'
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpdfgr210e\build\youtube_dl\YoutubeDL.py", line 1863, in process_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpdfgr210e\build\youtube_dl\YoutubeDL.py", line 1805, in dl
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpdfgr210e\build\youtube_dl\downloader\common.py", line 361, in download
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpdfgr210e\build\youtube_dl\downloader\http.py", line 61, in real_download
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpdfgr210e\build\youtube_dl\YoutubeDL.py", line 2166, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 502, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 685, in http_error_302
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden
...
<end of log>
```

---
Downloading live streams from Panda.TV suddenly returns error 403. Hope you can look into it. Thank you!

**Comment by shoujs** [comment]
try appending  `--cookie ` args. export your cookie from the browser. chrome plugin cookies.txt may help.

**Comment by remitamine** [comment]
fixed in v2017.11.15.
